### PR TITLE
KIALI-1112 Do a graph fit outside the batch operation to center the r…

### PR DESCRIPTION
**Pull request by josejulio**
…esult

This might also fix [KIALI-1060](https://issues.jboss.org/browse/KIALI-1060)

kiali-1112

### CODE REVIEW

File: src/components/CytoscapeGraph/CytoscapeGraph.tsx [hunk 1]
```diff
@@ -64,12 +64,14 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
...
   private cytoscapeReactWrapperRef: any;
   private updateLayout: boolean;
+  private refit: boolean;
   private resetSelection: boolean;
   private cy: any;
 
   constructor(props: CytoscapeGraphProps) {
     super(props);
     this.updateLayout = false;
+    this.refit = false;
```
Comment (jmazzitelli) [inline src/components/CytoscapeGraph/CytoscapeGraph.tsx]: Just a question - when is this.refit false again? I see it set to false in the constructor. But is this ever going to false again? I guess what this PR is doing is ensuring a refit is performed on every graph update now?

Comment (josejulio) [reply src/components/CytoscapeGraph/CytoscapeGraph.tsx]: In `processGraphUpdate` : https://github.com/kiali/kiali-ui/pull/501/files#diff-04b15b4602d6b9ccf1eb82d15e07535fR355

Comment (josejulio) [reply src/components/CytoscapeGraph/CytoscapeGraph.tsx]: Refit is performed on graph initialization and when there is a layout change: https://github.com/kiali/kiali-ui/pull/501/files#diff-04b15b4602d6b9ccf1eb82d15e07535fR325

Comment (jmazzitelli) [reply src/components/CytoscapeGraph/CytoscapeGraph.tsx]: ah... ok, i missed that. yeah, refit on every update would be bad, which is why i was confused :)

**Pull request merged**